uber/config.py

```diff
@@ -157,6 +157,42 @@ def BADGES_SOLD(self):
             return individuals + group_badges
 
     @property
+    def BADGES_LEFT_AT_CURRENT_PRICE(self):
+        """
+        Returns a string representing a rough estimate of how many badges are left at the current badge price tier.
+        """
+        if c.HARDCORE_OPTIMIZATIONS_ENABLED:
+            return 'Unknown'
+
+        current_price_tier = c.ORDERED_PRICE_LIMITS.index(c.BADGE_PRICE) if c.BADGE_PRICE in c.ORDERED_PRICE_LIMITS else -1
+        if current_price_tier != -1 and c.ORDERED_PRICE_LIMITS[current_price_tier] == c.ORDERED_PRICE_LIMITS[-1]\
+                or not c.ORDERED_PRICE_LIMITS:
+            if c.MAX_BADGE_SALES:
+                difference = c.MAX_BADGE_SALES - c.BADGES_SOLD
+            else:
+                return 'Limitless'
+        else:
+            for key, val in c.PRICE_LIMITS.items():
+                if c.ORDERED_PRICE_LIMITS[current_price_tier+1] == val:
+                    difference = key - c.BADGES_SOLD
+        if not difference:
+            return 'Unknown'
```

Comment:
Is there ever a condition where `difference` would be zero? That would mean "Sold Out".

Perhaps if this function were called from someplace other than the preregistration page, `difference` could definitely be zero.